__tests__/unit/_utils/get-toolkit-locations.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * __tests__/unit/_utils/get-toolkit-locations.js
+ * Test: js/_utils/_get-toolkit-locations.js
+ */
+'use strict';
+
+console.log = jest.fn();
```

Comment:
does this need to be here?

Reply:
Same as above, `jest.mock('@springernature/util-cli-reporter')` is more appropriate as we don't want the reporter outputting to the CLI